proto/v2/observation.proto

```diff
@@ -62,11 +62,9 @@ message FacetObservation {
 }
 
 message FacetFilter {
-  // List of variables that this facet filter applies to
-  repeated string variables = 1;
-  // When specified, only observations with provenance in this domain is
+  // When specified, only observations with provenance in these domains are
   // returned.
-  string domain = 2;
+  repeated string domains = 2;
   // When specified, only observations with this facet id are returned
   string facet_id = 3;
```

Comment:
i thought you want to keep repeated facet_id?

Reply:
chatted offline and changed to repeated